modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/causalitydatanodes/DistributionZoneCausalityDataNodesTest.java

```diff
@@ -595,41 +598,58 @@ void testEmptyDataNodesOnZoneCreationBeforeTopologyEvent() throws Exception {
 
     /**
      * Tests that data nodes for zones with different scale up/down configs are empty when creation of zones were before any
-     * topology event. In this test scenario we assume that initialisation of a zone was after the calling of the data nodes method.
+     * topology event. In this case, we check that the nodes are still empty relatively initial parameters even though a zone
+     * topology was already changed.
      */
     @ParameterizedTest
     @MethodSource("provideArgumentsOfDifferentTimersValue")
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-22833")
-    void testEmptyDataNodesOnZoneCreationBeforeTopologyEventAndZoneInitialisation(int scaleUp, int scaleDown) {
-        CountDownLatch latch = new CountDownLatch(1);
+    void testEmptyDataNodesOnZoneCreationBeforeTopologyEventAndZoneInitialisation(int scaleUp, int scaleDown) throws Exception {
```

Comment:
The test scenario is invalid. Please remove the test.

Reply:
Done.